app/_lib/moduleProps.tsx

```diff
@@ -21,75 +21,75 @@ export interface Module {
   description: string;
 }
 
-export const Tags = ['discover', 'trade', 'outfit'];
+export const Tags = ['Discover', 'Trade', 'Outfit'];
```

Comment:
@aslink87 Whats happening here, I can't work out what this change is doing

Reply:
I needed to make the tags in the desired case because they were no longer having their case changed in the component, so they rendered in lower. I've removed the lower case comparison check.